### PR TITLE
Tap architecture planning 

### PR DESCRIPTION
**Pull request by MarilynHorn**
IX Review. Changes to grammar, capitalization. Corrections of product names. Minor formatting changes.

Signed-off-by: hornm@vmware.com

### CODE REVIEW

**Comment by MarilynHorn** [comment]
PLEASE IGNORE the Following Commit Points:

https://github.com/vmware-tanzu-labs/tanzu-validated-solutions/pull/127/commits/84dd81ebbe9468b04be2e1f86b202de54c2908ba

https://github.com/vmware-tanzu-labs/tanzu-validated-solutions/pull/127/commits/7f2c34ab846d5f61d434d00e6efdaa864454589f

File: src/reference-designs/tap-architecture-planning.md [hunk 1]
```diff
@@ -120,17 +122,18 @@ excluded_packages:
  - workshops.learningcenter.tanzu.vmware.com
 ```
 ### View Cluster Requirements
-The View Cluster is designed to run the web applications for TAP. Specifically Tanzu Learning Center, Tanzu Application Portal GUI, and Tanzu API Portal.
-The View Cluster's requirements will be mostly driven by the respective applications that it will be running.
-Kubernetes requirements:
-* LoadBalancer for ingress controller (3 external IPs)
+The View Cluster is designed to run the web applications for TAP. specifically, Tanzu Learning Center, Tanzu Application Portal GUI, and Tanzu API Portal.
+The View Cluster's requirements are driven prinmarily by the respective applications that it will be running.
```
Comment (craigtracey) [inline src/reference-designs/tap-architecture-planning.md:126]: We need to fix the spelling for "primarily" here.

**Review by craigtracey — CHANGES_REQUESTED** [review]
Thanks for these changes, @MarilynHorn. 

There are two items we will need to resolve for getting this change merged. First, we need to address the misspelled "prinmarily" as this is failing spell check.

Second, we will need to get these commits signed by you. It looks like you added the text to the PR description, but these need to go into the commits themselves. I would recommend that we squash these changes into one commit, and sign it. 

To do so, the following should work:
`git reset --soft HEAD~3 && git commit -a -v -s`

As this will modify the history on your fork, you will need to push it with a `-f`

Please reach out if I can help with this.

**Comment by MarilynHorn** [comment]
Thanks, Craig.  I’m still very new to using GitHub so I’ll need some handholding. I appreciate your help with this.

For the spelling error, I made a suggested change in reply to your comment on GitHub.


For the squash command, I just made things worse because I ran it while I had a different branch checked out.  (I’m using SourceTree and the command lin.) Now I see uncommitted changes in SourceTree, and I get the following messages when I try to switch to the tap-architecture-planning branch.

git -c color.branch=false -c color.diff=false -c color.status=false -c diff.mnemonicprefix=false -c core.quotepath=false -c credential.helper=sourcetree reset -q --hard HEAD --

fatal: Unable to create '/Users/hornm/tanzu-validated-solutions/.git/index.lock': File exists.



Another git process seems to be running in this repository, e.g.

an editor opened by 'git commit'. Please make sure all processes

are terminated then try again. If it still fails, a git process

may have crashed in this repository earlier:

remove the file manually to continue.

Completed with errors, see above

What should I do now? I can send a screenshot if you need it.

Marilyn

From: Craig Tracey ***@***.***>
Date: Tuesday, March 29, 2022 at 10:43 AM
To: vmware-tanzu-labs/tanzu-validated-solutions ***@***.***>
Cc: Marilyn J. Horn (c) ***@***.***>, Mention ***@***.***>
Subject: Re: [vmware-tanzu-labs/tanzu-validated-solutions] Tap architecture planning (PR #127)

⚠ External Email

@craigtracey requested changes on this pull request.

Thanks for these changes, @MarilynHorn<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMarilynHorn&data=04%7C01%7Chornm%40vmware.com%7C09859e96c9e74c0346c908da119288b0%7Cb39138ca3cee4b4aa4d6cd83d9dd62f0%7C0%7C0%7C637841618316543502%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=5Qq91Ul9qK99fP933CRCFFPozlrv7hIXryafjx0vVp8%3D&reserved=0>.

There are two items we will need to resolve for getting this change merged. First, we need to address the misspelled "prinmarily" as this is failing spell check.

Second, we will need to get these commits signed by you. It looks like you added the text to the PR description, but these need to go into the commits themselves. I would recommend that we squash these changes into one commit, and sign it.

To do so, the following should work:
git reset --soft HEAD~3 && git commit -a -v -s

Please reach out if I can help with this.

________________________________

In src/reference-designs/tap-architecture-planning.md<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fvmware-tanzu-labs%2Ftanzu-validated-solutions%2Fpull%2F127%23discussion_r837559794&data=04%7C01%7Chornm%40vmware.com%7C09859e96c9e74c0346c908da119288b0%7Cb39138ca3cee4b4aa4d6cd83d9dd62f0%7C0%7C0%7C637841618316543502%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=YFfv6aygndce6EEdxvDELaiLz8t8Z34e0jZAZfRKMAU%3D&reserved=0>:

> @@ -120,17 +122,18 @@ excluded_packages:

  - workshops.learningcenter.tanzu.vmware.com

 ```

 ### View Cluster Requirements

-The View Cluster is designed to run the web applications for TAP. Specifically Tanzu Learning Center, Tanzu Application Portal GUI, and Tanzu API Portal.

-The View Cluster's requirements will be mostly driven by the respective applications that it will be running.

-Kubernetes requirements:

-* LoadBalancer for ingress controller (3 external IPs)

+The View Cluster is designed to run the web applications for TAP. specifically, Tanzu Learning Center, Tanzu Application Portal GUI, and Tanzu API Portal.

+The View Cluster's requirements are driven prinmarily by the respective applications that it will be running.

We need to fix the spelling for "primarily" here.

—
Reply to this email directly, view it on GitHub<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fvmware-tanzu-labs%2Ftanzu-validated-solutions%2Fpull%2F127%23pullrequestreview-924742320&data=04%7C01%7Chornm%40vmware.com%7C09859e96c9e74c0346c908da119288b0%7Cb39138ca3cee4b4aa4d6cd83d9dd62f0%7C0%7C0%7C637841618316543502%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=6q1SBZgYEZ3Hb4xJwoSVxElVb9bMtnKDv77UDP8Tryk%3D&reserved=0>, or unsubscribe<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAYBFEI2NSLM23YAATH2WWRTVCMJKJANCNFSM5R4QHFNA&data=04%7C01%7Chornm%40vmware.com%7C09859e96c9e74c0346c908da119288b0%7Cb39138ca3cee4b4aa4d6cd83d9dd62f0%7C0%7C0%7C637841618316543502%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=ACnlX3cs1AEJun9A%2FruCLGPub3xlqcH8HgnVPFPxXaM%3D&reserved=0>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

________________________________

⚠ External Email: This email originated from outside of the organization. Do not click links or open attachments unless you recognize the sender.

**Pull request merged**